### ISSUE TITLE
entrypoint checks only for '/var/www/html/user.config.php' not '/var/www/html/1.user.config.php

### ISSUE THREAD

**Issue by pixelchrome** [DeBaschdi/docker.solaranzeige#4]
The entrypoint script checks only for `/var/www/html/user.config.php`. If more than one device is configured, `1.user.config.php` should be used... Therefore the container will kick of the setup routine...

**Comment by DeBaschdi** [comment]
Is it really necesarry to check X.user.config.php ?
https://solaranzeige.de/phpBB3/viewtopic.php?p=17360#p17360

**Comment by pixelchrome** [comment]
In my case there was only `1.user.config.php` & `2.user.config.php` available (no `user.config.php`) and the docker image kicked off the setup routine and everything was gone. I was able to recover the data. Maybe it it was a glitch on my side but the entrypoint script IMHO only looks for `user.config.php` ... with out that it kicks off the setup.

I've just created a symbolic link `user.config.php -> 1.user.config.php` as a workaround.
